app/controllers/reviews_controller.rb

```diff
@@ -25,11 +30,22 @@ def edit
 
   def update
     review = Review.find(params[:review_id])
-    if review.update(review_params)
-      redirect_to "/shelters/#{review.shelter.id}"
-    else
+    review.update(review_params)
+
+    if review.rating == nil || review.title == nil
       flash[:notice] = "Please Fill In Title, Rating, and Content"
       redirect_to "/shelters/#{review.shelter.id}/reviews/#{review.id}/edit"
+
```

Comment:
Is there a conditional for if they don't enter any content? I think we need to add that too in case they don't fill in any content.